src/app/common/directives/oc-reorder/js/oc-reorder.modal.service.js

```diff
@@ -0,0 +1,121 @@
+angular.module('orderCloud')
+    .factory('ocReorderModal', ocReorderModalService)
+;
+
+function ocReorderModalService($q, toastr, $exceptionHandler, OrderCloudSDK, ocLineItems, $uibModal) {
```

Comment:
Think we can take "Modal" out of the service name? 
 I get that it's primary function is opening a modal but I feel like `oc-reorder.service.js` & `ocReorder` would be more globally applicable if someone was just using the service for `AddLineItemsToCart` or `GetValidLineItems`.

Just a suggestion, definitely not necessary.